website/content/docs/k8s/helm.mdx

```diff
@@ -152,22 +153,20 @@ Use these links to navigate to a particular top-level stanza.
       - `consulServerRole` ((#v-global-secretsbackend-vault-consulserverrole)) (`string: ""`) - The Vault role for the Consul server.
         The role must be connected to the Consul server's service account and
         have a policy with read capabilities for the following secrets:
+
         - gossip encryption key defined by `global.gossipEncryption.secretName`.
-        To discover the service account name of the Consul server, run
-            ```
-            helm template -s templates/server-serviceaccount.yaml <release-name> hashicorp/consul
-            ```
-        and check the name of `metadata.name`.
+          To discover the service account name of the Consul server, run
+          ` helm template -s templates/server-serviceaccount.yaml <release-name> hashicorp/consul `
+          and check the name of `metadata.name`.
 
       - `consulClientRole` ((#v-global-secretsbackend-vault-consulclientrole)) (`string: ""`) - The Vault role for the Consul client.
         The role must be connected to the Consul client's service account and
         have a policy with read capabilities for the following secrets:
+
         - gossip encryption key defined by `global.gossipEncryption.secretName`.
-        To discover the service account name of the Consul server, run
-            ```
-            helm template -s templates/client-daemonset.yaml <release-name> charts/consul
-            ```
-        and check the name of `metadata.name`.
+          To discover the service account name of the Consul server, run
+          ` helm template -s templates/client-daemonset.yaml <release-name> charts/consul `
+          and check the name of `metadata.name`.
```

Comment:
@t-eckert This needs to be using the `shell-session` component.  We also need the command to be in longform `--show-only` instead of `-s`

Reply:
Resolved in https://github.com/hashicorp/consul-k8s/pull/1010